Apps/Sandcastle/gallery/Star Burst.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Star burst overlapping billboards.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+                           baseUrl : '../../../Source',
+                           waitSeconds : 60
+                       });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <div id="zoomButtons"></div>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+"use strict";
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    selectionIndicator : false
+});
+
+// Add labels clustered at the same location
+var numBillboards = 28.0;
+for (var i = 0; i < numBillboards; ++i) {
+    var position = Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883);
+    viewer.entities.add({
+        position : position,
+        billboard : {
+            image : '../images/facility.gif',
+            scale : 2.5
+        },
+        label : {
+            text : 'Label' + i,
+            show : false
+        }
+    });
+}
+
+var scene = viewer.scene;
+
+// State saved across mouse click and move events
+var pickedEntities;
```

Comment:
Put this into an object literal.

In general, think about making this as easy as possible for users to copy and paste into their project.  Perhaps the whole thing could even go into a class that one day could be a candidate for core Cesium.